### ISSUE TITLE
openshift-ansible must not use {cluster}policy{binding} objects

### ISSUE THREAD

**Issue by enj** [openshift/openshift-ansible#4880]
```
TASK [openshift_manageiq : Configure role/user permissions] ********************
task path: /usr/share/ansible/openshift-ansible/roles/openshift_manageiq/tasks/main.yaml:43
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: __main__.OpenShiftCLIError: Could not retrieve policybindings
failed: [localhost] (item={u'resource_kind': u'role', u'resource_name': u'admin', u'user': u'management-admin'}) => {
    "failed": true, 
    "generated_timestamp": "2017-07-26 12:17:03.930452", 
    "item": {
        "resource_kind": "role", 
        "resource_name": "admin", 
        "user": "management-admin"
    }, 
    "module_stderr": [
        "Traceback (most recent call last):", 
        "  File \"/tmp/ansible_4y6czs/ansible_module_oc_adm_policy_user.py\", line 2171, in <module>", 
        "    main()", 
        "  File \"/tmp/ansible_4y6czs/ansible_module_oc_adm_policy_user.py\", line 2162, in main", 
        "    results = PolicyUser.run_ansible(module.params, module.check_mode)", 
        "  File \"/tmp/ansible_4y6czs/ansible_module_oc_adm_policy_user.py\", line 2117, in run_ansible", 
        "    results = policyuser.exists()", 
        "  File \"/tmp/ansible_4y6czs/ansible_module_oc_adm_policy_user.py\", line 2056, in exists", 
        "    return self.exists_role_binding()", 
        "  File \"/tmp/ansible_4y6czs/ansible_module_oc_adm_policy_user.py\", line 2022, in exists_role_binding", 
        "    bindings = self.policybindings", 
        "  File \"/tmp/ansible_4y6czs/ansible_module_oc_adm_policy_user.py\", line 1969, in policybindings", 
        "    raise OpenShiftCLIError('Could not retrieve policybindings')", 
        "__main__.OpenShiftCLIError: Could not retrieve policybindings"
    ], 
    "module_stdout": [], 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: __main__.OpenShiftCLIError: Could not retrieve policybindings
failed: [localhost] (item={u'resource_kind': u'role', u'resource_name': u'management-infra-admin', u'user': u'management-admin'}) => {
    "failed": true, 
    "generated_timestamp": "2017-07-26 12:17:04.322095", 
    "item": {
        "resource_kind": "role", 
        "resource_name": "management-infra-admin", 
        "user": "management-admin"
    }, 
    "module_stderr": [
        "Traceback (most recent call last):", 
        "  File \"/tmp/ansible_igVM2s/ansible_module_oc_adm_policy_user.py\", line 2171, in <module>", 
        "    main()", 
        "  File \"/tmp/ansible_igVM2s/ansible_module_oc_adm_policy_user.py\", line 2162, in main", 
        "    results = PolicyUser.run_ansible(module.params, module.check_mode)", 
        "  File \"/tmp/ansible_igVM2s/ansible_module_oc_adm_policy_user.py\", line 2117, in run_ansible", 
        "    results = policyuser.exists()", 
        "  File \"/tmp/ansible_igVM2s/ansible_module_oc_adm_policy_user.py\", line 2056, in exists", 
        "    return self.exists_role_binding()", 
        "  File \"/tmp/ansible_igVM2s/ansible_module_oc_adm_policy_user.py\", line 2022, in exists_role_binding", 
        "    bindings = self.policybindings", 
        "  File \"/tmp/ansible_igVM2s/ansible_module_oc_adm_policy_user.py\", line 1969, in policybindings", 
        "    raise OpenShiftCLIError('Could not retrieve policybindings')", 
        "__main__.OpenShiftCLIError: Could not retrieve policybindings"
    ], 
    "module_stdout": [], 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: __main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings
failed: [localhost] (item={u'resource_kind': u'cluster-role', u'resource_name': u'cluster-reader', u'user': u'system:serviceaccount:management-infra:management-admin'}) => {
    "failed": true, 
    "generated_timestamp": "2017-07-26 12:17:04.718953", 
    "item": {
        "resource_kind": "cluster-role", 
        "resource_name": "cluster-reader", 
        "user": "system:serviceaccount:management-infra:management-admin"
    }, 
    "module_stderr": [
        "Traceback (most recent call last):", 
        "  File \"/tmp/ansible_wMAGuy/ansible_module_oc_adm_policy_user.py\", line 2171, in <module>", 
        "    main()", 
        "  File \"/tmp/ansible_wMAGuy/ansible_module_oc_adm_policy_user.py\", line 2162, in main", 
        "    results = PolicyUser.run_ansible(module.params, module.check_mode)", 
        "  File \"/tmp/ansible_wMAGuy/ansible_module_oc_adm_policy_user.py\", line 2117, in run_ansible", 
        "    results = policyuser.exists()", 
        "  File \"/tmp/ansible_wMAGuy/ansible_module_oc_adm_policy_user.py\", line 2053, in exists", 
        "    return self.exists_role_binding()", 
        "  File \"/tmp/ansible_wMAGuy/ansible_module_oc_adm_policy_user.py\", line 2020, in exists_role_binding", 
        "    bindings = self.clusterpolicybindings", 
        "  File \"/tmp/ansible_wMAGuy/ansible_module_oc_adm_policy_user.py\", line 1979, in clusterpolicybindings", 
        "    raise OpenShiftCLIError('Could not retrieve clusterpolicybindings')", 
        "__main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings"
    ], 
    "module_stdout": [], 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
ok: [localhost] => (item={u'resource_kind': u'scc', u'resource_name': u'privileged', u'user': u'system:serviceaccount:management-infra:management-admin'}) => {
    "changed": false, 
    "generated_timestamp": "2017-07-26 12:17:05.102418", 
    "item": {
        "resource_kind": "scc", 
        "resource_name": "privileged", 
        "user": "system:serviceaccount:management-infra:management-admin"
    }, 
    "present": "present"
}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: __main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings
failed: [localhost] (item={u'resource_kind': u'cluster-role', u'resource_name': u'system:image-puller', u'user': u'system:serviceaccount:management-infra:inspector-admin'}) => {
    "failed": true, 
    "generated_timestamp": "2017-07-26 12:17:05.535673", 
    "item": {
        "resource_kind": "cluster-role", 
        "resource_name": "system:image-puller", 
        "user": "system:serviceaccount:management-infra:inspector-admin"
    }, 
    "module_stderr": [
        "Traceback (most recent call last):", 
        "  File \"/tmp/ansible_HyBTwi/ansible_module_oc_adm_policy_user.py\", line 2171, in <module>", 
        "    main()", 
        "  File \"/tmp/ansible_HyBTwi/ansible_module_oc_adm_policy_user.py\", line 2162, in main", 
        "    results = PolicyUser.run_ansible(module.params, module.check_mode)", 
        "  File \"/tmp/ansible_HyBTwi/ansible_module_oc_adm_policy_user.py\", line 2117, in run_ansible", 
        "    results = policyuser.exists()", 
        "  File \"/tmp/ansible_HyBTwi/ansible_module_oc_adm_policy_user.py\", line 2053, in exists", 
        "    return self.exists_role_binding()", 
        "  File \"/tmp/ansible_HyBTwi/ansible_module_oc_adm_policy_user.py\", line 2020, in exists_role_binding", 
        "    bindings = self.clusterpolicybindings", 
        "  File \"/tmp/ansible_HyBTwi/ansible_module_oc_adm_policy_user.py\", line 1979, in clusterpolicybindings", 
        "    raise OpenShiftCLIError('Could not retrieve clusterpolicybindings')", 
        "__main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings"
    ], 
    "module_stdout": [], 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
ok: [localhost] => (item={u'resource_kind': u'scc', u'resource_name': u'privileged', u'user': u'system:serviceaccount:management-infra:inspector-admin'}) => {
    "changed": false, 
    "generated_timestamp": "2017-07-26 12:17:05.948200", 
    "item": {
        "resource_kind": "scc", 
        "resource_name": "privileged", 
        "user": "system:serviceaccount:management-infra:inspector-admin"
    }, 
    "present": "present"
}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: __main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings
failed: [localhost] (item={u'resource_kind': u'cluster-role', u'resource_name': u'self-provisioner', u'user': u'system:serviceaccount:management-infra:management-admin'}) => {
    "failed": true, 
    "generated_timestamp": "2017-07-26 12:17:06.379698", 
    "item": {
        "resource_kind": "cluster-role", 
        "resource_name": "self-provisioner", 
        "user": "system:serviceaccount:management-infra:management-admin"
    }, 
    "module_stderr": [
        "Traceback (most recent call last):", 
        "  File \"/tmp/ansible_JQGNcP/ansible_module_oc_adm_policy_user.py\", line 2171, in <module>", 
        "    main()", 
        "  File \"/tmp/ansible_JQGNcP/ansible_module_oc_adm_policy_user.py\", line 2162, in main", 
        "    results = PolicyUser.run_ansible(module.params, module.check_mode)", 
        "  File \"/tmp/ansible_JQGNcP/ansible_module_oc_adm_policy_user.py\", line 2117, in run_ansible", 
        "    results = policyuser.exists()", 
        "  File \"/tmp/ansible_JQGNcP/ansible_module_oc_adm_policy_user.py\", line 2053, in exists", 
        "    return self.exists_role_binding()", 
        "  File \"/tmp/ansible_JQGNcP/ansible_module_oc_adm_policy_user.py\", line 2020, in exists_role_binding", 
        "    bindings = self.clusterpolicybindings", 
        "  File \"/tmp/ansible_JQGNcP/ansible_module_oc_adm_policy_user.py\", line 1979, in clusterpolicybindings", 
        "    raise OpenShiftCLIError('Could not retrieve clusterpolicybindings')", 
        "__main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings"
    ], 
    "module_stdout": [], 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: __main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings
failed: [localhost] (item={u'resource_kind': u'cluster-role', u'resource_name': u'hawkular-metrics-admin', u'user': u'system:serviceaccount:management-infra:management-admin'}) => {
    "failed": true, 
    "generated_timestamp": "2017-07-26 12:17:06.854999", 
    "item": {
        "resource_kind": "cluster-role", 
        "resource_name": "hawkular-metrics-admin", 
        "user": "system:serviceaccount:management-infra:management-admin"
    }, 
    "module_stderr": [
        "Traceback (most recent call last):", 
        "  File \"/tmp/ansible_ytFzgk/ansible_module_oc_adm_policy_user.py\", line 2171, in <module>", 
        "    main()", 
        "  File \"/tmp/ansible_ytFzgk/ansible_module_oc_adm_policy_user.py\", line 2162, in main", 
        "    results = PolicyUser.run_ansible(module.params, module.check_mode)", 
        "  File \"/tmp/ansible_ytFzgk/ansible_module_oc_adm_policy_user.py\", line 2117, in run_ansible", 
        "    results = policyuser.exists()", 
        "  File \"/tmp/ansible_ytFzgk/ansible_module_oc_adm_policy_user.py\", line 2053, in exists", 
        "    return self.exists_role_binding()", 
        "  File \"/tmp/ansible_ytFzgk/ansible_module_oc_adm_policy_user.py\", line 2020, in exists_role_binding", 
        "    bindings = self.clusterpolicybindings", 
        "  File \"/tmp/ansible_ytFzgk/ansible_module_oc_adm_policy_user.py\", line 1979, in clusterpolicybindings", 
        "    raise OpenShiftCLIError('Could not retrieve clusterpolicybindings')", 
        "__main__.OpenShiftCLIError: Could not retrieve clusterpolicybindings"
    ], 
    "module_stdout": [], 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
 [WARNING]: Could not create retry file '/usr/share/ansible/openshift-
ansible/playbooks/byo/openshift-cluster/upgrades/v3_6/upgrade.retry'.
[Errno 13] Permission denied: u'/usr/share/ansible/openshift-
ansible/playbooks/byo/openshift-cluster/upgrades/v3_6/upgrade.retry'

PLAY RECAP *********************************************************************
localhost                  : ok=433  changed=46   unreachable=0    failed=1   

++ export status=FAILURE
++ status=FAILURE
+ set +o xtrace
########## FINISHED STAGE: FAILURE: UPDATE ORIGIN TO RELEASE [00h 07m 41s] ##########
Build step 'Execute shell' marked build as failure
[PostBuildScript] - Execution post build scripts.
```

@openshift/security

@sdodson the `oc` CLI wrappers need to be updated to use `{cluster}role{bindings}`

This blocks openshift/origin/pull/15021 which needs to get into origin ASAP

**Comment by sdodson** [comment]
@kwoodson and @enj can you two look at this together tomorrow?

**Comment by enj** [comment]
Fixed by #4933
